realm/realm-library/src/androidTest/java/io/realm/LinkingObjectsUnmanagedTests.java

```diff
@@ -109,9 +86,11 @@ public void copyToAndFromRealm() {
         assertEquals(1, parents.size());
         assertEquals(parent, parents.first());
 
+        // backlinks fields in unmanaged object created from managed object are null.
+
         unmanagedChild = realm.copyFromRealm(child);
         assertEquals(unmanagedChild.getFieldId(), 1);
-        assertEquals(new AllJavaTypes().getObjectParents(), unmanagedChild.getObjectParents());
+        assertNull(unmanagedChild.getObjectParents());
 
         RealmResults<AllJavaTypes> queryResults = realm.where(AllJavaTypes.class).equalTo("fieldId", 1).findAll();
```

Comment:
I am unclear on why it is a good idea to change the semantics of these tests.  Did we change the semantics of backlinks???